### DISCUSSION TITLE
Upload gif files into markdown cells of .ipynb

### DISCUSSION THREAD

**Discussion by yannis-bit** [Part-II-Computational-Physics/.github#18] [General]
I have tried to upload gif of the animations, using 
![ txt ]( sample.gif ) 

They are shown correctly locally in vscode. But in the github preview, bad image sign was displayed. (I have uploaded the gif files onto the repository as well). Any suggestion?

**Comment by NathanMagnan** [comment]
Hey,

I can indeed not see the gif files. 
But that's allright,we don't really need gifs.
Just explain in one or two sentences what happens in the video (like if you made a lab report)
